src/MgvLib.sol

```diff
@@ -34,6 +34,16 @@ library OLLib {
   function flipped(OLKey memory olKey) internal pure returns (OLKey memory) {
     return OLKey(olKey.inbound, olKey.outbound, olKey.tickSpacing);
   }
+
+  // Convert tick to bin according to olKey's tickSpacing
+  function bin(OLKey memory olKey, Tick _tick) internal pure returns (Bin) {
```

Comment:
Unsurprisingly, I'm not a fan of hiding that this is a lossy conversion.

If do you choose to go that route, it should be consistent, so `toNearestBin` should be `toBin`. But again, I don't think we should do that 😅

Reply:
fixed in 8ef390398905f50d1d11d6462ed559fe86b0a90b